### DISCUSSION TITLE
Rotate texture doesn't work.

### DISCUSSION THREAD

**Discussion by WDavidC** [pixijs/pixijs#7643] [General]
Hey, i'm trying to create an animation of shoe walking in a direction by rotating the texture but it's seems to not work the way i did.
I'm trying to make a array of texture of the shoe png and then trying to rotate the texture.

`    
              let textures = [];

    for(let i = 0; i < 8; i++){
	const texture = new Texture.from("shoe.png");
        texture.rotation = (rotation deg);
	textures.push(texture);
    }
      let shl = new AnimatedSprite(textures);`

Is there any better way to do it ?

**Comment by ivanpopelyshev** [comment]
1. you have to clone the texture first, `from` is caching, its not creating new one.
2. `texture.rotate` is 2 for 90 degrees. https://github.com/pixijs/pixijs/blob/dev/packages/core/src/textures/Texture.ts#L639

**Reply by WDavidC** [reply]
Is it only 90 degrees rotation available ? Like if a 8 frames on each frame i gotta rotate at some degree.

**Reply by ivanpopelyshev** [reply]
texture rotate UV's. Can you tell me which UV's on sprite corners to use for 60 degrees? :)

if you want sprite rotation, sorry, you have to make that animation yourself. Even more, pixi doesnt have animation manager, nor tweens, you have to take GSAP or code something yourself.

**Reply by ivanpopelyshev** [reply]
also, about AnimatedSprite - its basic class that people usually copy and make their own animated sprites based on it. It has very limited functionality.
